pandas/core/internals.py

```diff
@@ -1929,6 +1910,7 @@ def should_store(self, value):
 
 
 class DatetimeLikeBlockMixin(object):
+    freq = None  # compat with Datetimelike Index subclasses
```

Comment:
put comments above this, but think this comment is out of place, meaning that unless you have internal knowledge of the codebase you won't have any idea what this is, rather put a 1-liner that explains